docs/guides/primitive-input-output.ipynb

```diff
@@ -356,6 +356,28 @@
     "print(f'The expectation values measured from this PUB are: \\n{result[0].data.evs}')"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "40a9f976-fffa-48fb-a81a-26446d5a65a4",
+   "metadata": {},
+   "source": [
+    "#### How the Estimator calculates error\n",
+    "\n",
+    "In addition to the estimate of the mean of the observables passed in the input PUBs (the `evs` field of the `DataBin`), the Estimator also attempts to deliver an estimate of the error associated with those expectation values. All estimator queries will populate the `stds` field with a quantity like the standard error of the mean for each expectation value, but some error mitigation options produce additional information, such as `ensemble_standard_error`.\n",
+    "\n",
+    "Consider a single observable $\\mathcal{O}$. In the absence of [ZNE](/guides/error-mitigation-and-suppression-techniques#zero-noise-extrapolation-zne), you can think of each shot of the Estimator execution as providing a point estimate of the expectation value $\\langle \\mathcal{O} \\rangle$. If the pointwise estimates are in a vector `Os`, then the value returned in `ensemble_standard_error` is equivalent to the following (in which $\\sigma_{\\mathcal{O}}$ is the standard deviation of the expectation value estimate and $N$ is the number of shots):\n",
+    "\n",
+    "$$\\frac{ \\sigma_{\\mathcal{O}} }{ \\sqrt{N} },$$\n",
+    "\n",
+    "which treats all shots as part of a single ensemble. If you requested gate [twirling](/guides/error-mitigation-and-suppression-techniques#pauli-twirling) (`twirling.enable_gates = True`), you can sort the pointwise estimates of $\\langle \\mathcal{O} \\rangle$ into sets that share a common twirl. Call these sets of estimates `O_twirls`, and there are `num_randomizations` (number of twirls) of them. Then `stds` is the standard error of the mean of `O_twirls`, as in\n",
```

Comment:
I've linked out to the sections on Pauli twirling and ZNE in the [Error mitigation and suppression techniques](/guides/error-mitigation-and-suppression-techniques) guide, unless there's a better source to point to @kaelynj @blakejohnson ?

Reply:
Seems ok to me.